src/components/Settings/components/Button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const Button = ({ children, onClick, disabled, type, className = '' }) => {
+	const btnClass = classNames('button', className, {
+		'button-disabled': disabled,
+		'button-delete': type === 'delete',
```

Comment:
Wat zijn de classNames `button-disabled` en `button-delete` ?

Reply:
<img width="375" alt="Screenshot 2022-05-12 at 13 37 34" src="https://user-images.githubusercontent.com/7559956/168066367-2ad4eb46-bd55-4a8c-a437-7d68290f290b.png">
Graag ook mooi uitlijnen ;) 
En zelfde stijl van knoppen gebruiken

Reply:
de classnames mochten idd weg, was copy-paste uit het extensieproject dus stond dat er nog in. opgelost, style aangepast en uitlijning toegepast.